src/_tests_/Home.test.js

```diff
@@ -0,0 +1,31 @@
+import { screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import renderer from 'react-test-renderer';
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter as Router } from 'react-router-dom';
+import store from '../redux/store';
+import Home from '../pages/Home';
+import renderProvider from '../redux/Stock-Screener/stockScreener-test';
+
+it('snapshots', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <Router>
+          <Home />
+        </Router>
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] One of the tests is failing, let's make sure all the tests are passing. ✍️

![Screenshot from 2023-03-22 19-15-19](https://user-images.githubusercontent.com/17243325/226973065-fd9f0b1e-f037-48cd-ab80-1dd52d661584.png)

Reply:
thanks for your good and valuable feedback.